ui/css/index.scss

```diff
@@ -1,7 +1,7 @@
 /*
   MetaMask design system imports
 */
-@import './reset.scss';
```

Comment:
Tailwind CSS includes its own reset, so we can remove this one. In fact, there's a specificity issue where the reset is overriding Tailwind class names so we *need* to replace it in favor of the tailwind reset